### ISSUE TITLE
Fix all occurrences quick action

### ISSUE THREAD

**Issue by josh-degraw** [DotNetAnalyzers/IDisposableAnalyzers#177]
I recently added this analyzer package to some relatively large code bases, and it has helped me find a handful of bugs. My problem is I also found out this way that there were some types that were being used extensively across the code base that implement `IDisposable`, previously unkown to me. 

So there are hundreds, if not thousands, of instances of simply creating an instance of this object and using it without disposing it, when it should be disposed.

Thankfully my specific case doesn't _seem_ to be a critical issue, but I would like to plug this hole ASAP, and having to go through individually has proven prohibitively time consuming.

Being able to apply at least the `Add using to end of block` quick action to all of the instances in a file for example would be incredibly helpful.

**Comment by JohanLarsson** [comment]
Adding fixall support will be easy, should just be one line of code in most cases.
The design is a bit deliberate here though, my thinking was that doing them one by one lets us review things so we don't introduce new bugs.
But on the other hand one can see it as an action the developer actively chooses so in that regard exposing the functionality and making it the devs responsibility to review makes sense.

**Comment by JohanLarsson** [comment]
If you are new to this analyzer a word of caution is in order: don't trust the analyzer blindly. Figuring out ownership for all cases is a pretty tricky problem and the analyzer is not perfect. So treat the reported warnings with a dose of skepticism and write issues if you find places where I'm dumbing analysis.

**Comment by josh-degraw** [comment]
@JohanLarsson Yeah I have definitely found cases where it makes _way_ more sense to do it one by one, but in my specific use case it makes sense: it's in a project whose sole purpose is creating and calling these objects that implement `IDisposable`, and just applying that fix across the project where I know all usages of it conform with an appropriate top-level `using` block would make this process go a lot smoother.

I will say that I can see it making sense to only add this for the fix that wraps the whole block in a `using`, because I have ran into strange behavior with the other fix that tries to optimistically dispose.

**Comment by JohanLarsson** [comment]
>  I have ran into strange behavior with the other fix that tries to optimistically dispose.
Never hesitate to write issues, nothing is too small. And if you don't have time to write a clean repro just throw in a huge code dump or a stack trace. Bad issue is better than no issue.

**Comment by JohanLarsson** [comment]
I'll try to find time to add fix all support for the add using to end of scope. Note that ideally a using should be minimal. I added this fix mostly as a convenience when used in tests. Resharper does a nice job at adding using with minimal scope so never implemented that. IIRC they also added add using to Roslyn so there may be built in support that maybe has more fix all options.

There is a minor problem with fixing and releasing. I started rewriting things but got interrupted so master is not in perfect state, should have used a branch. I'll try to find time to put us back in releasable state.
